src/containers/DataPackageView.js

```diff
@@ -140,21 +145,30 @@ export class DataPackageViewContainer extends React.Component {
 
   render() {
     return (
-      <DataPackagePanel specs={this.state.specs}/>
+      <div className="col-lg-10">
+        <DataPackagePanel specs={this.state.specs}/>
+        <ReactMarkdown source={this.props.metadata.readme}/>
+      </div>
     );
   }
 }
 
-DataPackageViewContainer.propTypes = {
-  dataPackageActions: PropTypes.object
+DataPackageView.propTypes = {
+  dataPackageActions: PropTypes.object,
+  publisherName: PropTypes.string.isRequired,
+  packageName: PropTypes.string.isRequired,
+  metadata: PropTypes.object
 };
 
 function mapStateToProps (state, ownProps) {
-  const { datapackage, resources } = state.dpr;
+  const { datapackage, resources, metadata } = state.dpr;
 
   return {
     datapackage: datapackage,
-    resources: resources
+    resources: resources,
+    metadata: metadata,
```

Comment:
Have to find a better name. This attribute will hold readme and other stuff (e.g title, description) , only read me will not be enough.